docs/user-guide/custom-themes.md

```diff
@@ -118,6 +118,7 @@ favicon           | [site_favicon]        |
 page_description  | [site_description]    |
 repo_url          | [repo_url]            |
 repo_name         | [repo_name]           |
+edit_uri          | [edit_uri]            |
```

Comment:
Not a problem. That will be getting a refactor soon, which should clear things up a bit.